src/gt4sd/frameworks/enzeptional/processing.py

```diff
@@ -232,3 +324,39 @@ def reconstruct_sequence_with_mutation_range(
         mutated_sequence_offset = end + 1
     mutated_sequence += sequence[end + 1 :]
     return mutated_sequence
+
+
+def selection(scores: List[Dict[str, Any]], k: int = 20) -> List[Any]:
+    """Select the top k mutated sequences based on the score
+
+    Args:
+        scores: dictionary containing sequences and scores.
+        K: number of top sequences to return
+
+    return:
+        retuen the top K sequences
+
+    """
+
+    res = list(sorted(scores, key=lambda d: d["score"], reverse=True))[:k]
+
+    return res
+
+
+def crossover(p1: str, p2: str) -> Tuple[str, str]:
```

Comment:
this should leave in `genetic_algorithm.py`

Reply:
those `p1` and `p2` do not help as variable names, maybe `a_sequence` and `another_sequence` are better